src/test/java/seedu/address/logic/commands/DeletePersonFromModuleCommandTest.java

```diff
@@ -108,6 +109,21 @@ public void execute_deletePersonFromModuleWhenPersonExistsInAddressBookButNotInM
         assertEquals(expectedPersonSet, modelStub.module.getPersons()); // Module's set of persons remains unchanged.
     }
 
+    @Test
+    public void execute_deletePersonFromModuleWhenModuleDoesNotExists_throwCommandException() {
```

Comment:
Sorry for the minor nitpick. Grammatical error here?

```suggestion
    public void execute_deletePersonFromModuleWhenModuleDoesNotExist_throwCommandException() {
```

Reply:
Good catch!